doc/source/contributing.rst

```diff
@@ -20,16 +20,26 @@ development mode, run:
     python -m pip install --upgrade pip
     pip install -e .
 
+
+Add new antennas
+----------------
+TBD
```

Comment:
You might consider filling that (or at least opening an issue as futur work)

Reply:
#259